.github/workflows/actions/download/action.yml

```diff
@@ -29,6 +29,11 @@ runs:
       run: |
         curl -LSfs https://archive.apache.org/dist/jena/binaries/apache-jena-fuseki-4.5.0.tar.gz -o RACK/rack-box/files/fuseki.tar.gz
 
+    - name: Download Apache Jena libraries
+      shell: bash
+      run: |
+        curl -LSfs https://archive.apache.org/dist/jena/binaries/apache-jena-4.5.0.tar.gz -o RACK/rack-box/files/apache-jena.tar.gz
```

Comment:
We currently use Fuseki 4.6.1, not 4.5.0, so you should use 4.6.1 as well.  You also might as well shorten the name to jena.tar.gz since we drop the apache- from fuseki.tar.gz too.

Reply:
Oh it is already changed... Weird, I didn't get these changes. Will fix!